Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:alpine
```

Add a particular version of node.
```suggestion
FROM node:20-alpine
```